dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/threading/LibertyJaxWsAutomaticWorkQueueImpl.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2019 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```

Comment:
should be `2019, 2020`
we are supposed to preserve the original date

Reply:
Thanks to my multiple commits its actually hard to tell that this is a new to JAX-WS file. So I think 2020 is still applicable.